examples/tokenvm/DEVNETS.md

```diff
@@ -92,14 +91,17 @@ Now we can spin up a new network of 6 nodes with some defaults:
 --network-name custom \
 --avalanchego-release-tag v1.10.3 \
 --create-dev-machine \
---keys-to-generate 5
+--keys-to-generate 5 \
+--profile-name <AWS_PROFILE_NAME>
 ```
 
 The `default-spec` (and `apply`) command only provisions nodes, not Custom VMs.
 The Subnet and Custom VM installation are done via `install-subnet-chain` sub-commands that follow.
 
 *NOTE*: The `--auto-regions 3` flag only lists the pre-defined regions and evenly distributes anchor and non-anchor nodes across regions in the default spec output. It is up to the user to change the regions and spec file based on the regional resource limits (e.g., pick the regions with higher EIP limits). To see what regions available in your AWS account, simply run the [`aws account list-regions`](https://docs.aws.amazon.com/cli/latest/reference/account/list-regions.html) command, or look at the dropdown menu of your AWS console.
 
+*NOTE*: Multiple regions can alternatively be selected by providing a comma-separated list of regions to the `--regions` flag. This will also spin up a dev machine in each region selected, if `--create-dev-machine` is enabled.
```

Comment:
How does this interact with the regions passed into the instance-types map?

Reply:
> How does this interact with the regions passed into the instance-types map?

avalancheup uses the `--regions` flag when determining in which regions to spin up dev machines specifically. The dev machine can be further customized using the `dev_machine` field on the spec. There is a `dev_machine.instance_types` field to select specific types. 

The `instance-type` is used for nodes in the subnet specifically. If you use `--regions` only then avalancheup generates an `instance-type` map under the hood using the default EC2 instances in each specific region. 

My impression is that `--regions` and --instance-types` can be used together without conflict.  The dev machines regions should be a subset of the overall regions selected.